### PR TITLE
update man page

### PR DESCRIPTION
**Pull request by undef21**
- add info about files/soundfonts directory;
- rephrase a sentence about music.

### CODE REVIEW

**Review by ihhub — APPROVED** [review]

File: docs/fheroes2.6.in [hunk 1]
```diff
@@ -42,14 +42,17 @@ Additional non-original assets from fheroes2 project.
 .B files/lang/*.mo
 Translations from fheroes2 project.
 .TP
+.B files/soundfonts/*
+Additional SoundFont files for MIDI synthesier. sf2 and sf3 formats are supported.
```
Comment (oleg-derevenetz) [inline docs/fheroes2.6.in:46]: It looks like there is a typo: `synthesier` -> `synthesizer`.

**Review by oleg-derevenetz — CHANGES_REQUESTED** [review]
Hi @undef21 LGTM in general, just one typo that needs to be fixed :)

File: docs/fheroes2.6.in (hunk 1 above)
Comment (undef21) [reply docs/fheroes2.6.in]: Thanks. Fixed.

**Review by oleg-derevenetz — APPROVED** [review]

**Comment by ihhub** [comment]
@undef21 , thank you very much for these changes!

**Pull request merged**